kvrocks.conf

```diff
@@ -329,6 +329,13 @@ json-max-nesting-depth 1024
 # Default: json
 json-storage-format json
 
+# The write buffer size of minor column families those infrequently be used
+# including pubsub, propagte, zset_score, stream and search. 
+# It will reduce the memory usage in some scenarios.
```

Comment:
I didn't check the communication previously, why this uses KB but major uses MB?

Reply:
> We can keep the write_buffer_size as a relatively smaller one like 16MiB/32MiB instead of 1 byte.

Yes, it will be better to use MiB to keep consistent with the option `write_buffer_size`.